scripts/generate-app-config/generatePagesConfig.js

```diff
@@ -136,6 +136,91 @@ const buildPageConfig = (filePaths, generatePagesOptions, namespace) => (
   }, {})
 );
 
+const getScreenshotPatterns = generatePagesOptions => (
+  generatePagesOptions.searchPatterns.reduce((acc, { root }) => {
+    const rootPath = root.replace(/[\\]/g, '/');
+    acc.push({
+      pattern: `${rootPath}/packages/*/tests/wdio/__snapshots__/reference/**/*.png`,
+      // build out a regex for the entrypoint mask.
+      entryPoint: `${rootPath}/packages/[^/]*/tests/wdio/__snapshots__/reference/`,
+    });
+    acc.push({
+      pattern: `${rootPath}/tests/wdio/__snapshots__/reference/**/*.png`,
+      // build out a regex for the entrypoint mask.
+      entryPoint: `${rootPath}/tests/wdio/__snapshots__/reference/`,
+    });
+
+    return acc;
+  }, [])
+);
+
+const createEvidenceParent = packageNamespace => (
+  {
+    name: startCase(packageNamespace),
+    path: `/${kebabCase(packageNamespace)}`,
+    group: '',
+    pages: {},
+  }
+);
+
+const createEvidencePage = (packageNamespace, name) => (
+  {
+    name: startCase(name),
+    path: packageNamespace ? `/${kebabCase(packageNamespace)}/${kebabCase(name)}` : `/${kebabCase(name)}`,
+    group: '',
+    content: {},
+    type: 'screenshot',
```

Comment:
I wonder if a different type would be desirable? At some point would we pull in jest coverage results or a collective test coverage evidence?